src/content/en/tools/workbox/reference-docs/latest/jsdoc.css

```diff
@@ -65,3 +65,7 @@ p.symbol-index-name {
 h2.symbol-header {
   margin: 36px 0px 0px 0px;
 }
+
+table.function.param tr {
```

Comment:
Kind of weird that this was added in? But seems harmless.

Reply:
Yeah, noticed that too. Maybe a new version of jsdocs?

Reply:
Hmmm, guess not: https://github.com/jsdoc3/jsdoc/releases